Makefile.toml

```diff
@@ -231,6 +233,7 @@ done
 rm -f ${BUILDSYS_TOOLS_DIR}/bin/buildsys
 for ext in rpm tar lz4 img ; do
   rm -f ${BUILDSYS_OUTPUT_DIR}/*.${ext}
+  rm -f ${BUILDSYS_PACKAGES_DIR}/*.${ext}
```

Comment:
Seems like we could remove rpm from the list for OUTPUT_DIR and only remove RPMs from PACKAGES_DIR; anything else wouldn't be ours.

Reply:
Sure - that makes sense. I was going for the "hammer and everything is nails approach". :)